### ISSUE TITLE
Extra Leaves and Short Grass not affect by shader

### ISSUE THREAD

**Issue by Fortuna67** [octarine-noise/BetterFoliage#219]
Please fix the extra leaves and short grass when Shader wind is enable with Better Foliage 2.2.0

![2018-12-22_14 42 55](https://user-images.githubusercontent.com/44172841/50375082-4ccc5a80-05f8-11e9-9ea6-019158e8f805.png)
![2018-12-22_14 42 36](https://user-images.githubusercontent.com/44172841/50375083-4ccc5a80-05f8-11e9-996d-04d621ca6da1.png)

**Comment by QuerrixTU** [comment]
totaly agree i wanna make some video but i have the same probleme...

**Comment by danibot-dz** [comment]
Yeah! I totaly agree too, i have the same problem. I refuse to make screenshots without your mod, please...
![image](https://user-images.githubusercontent.com/40378258/53464601-8b7ff600-3a29-11e9-995f-0f7d1302d550.png)

**Comment by Rexhunter99** [comment]
This is still an issue. None of the Better Foliage added geometry is being manipulated by any shader packs in 1.12.2.  Even with the checkbox ticked for shader support in the settings of Better Foliage.  It looks extremely odd to have the leaves block morphing 'In the wind' but the extra leaves textures to not do so, also the small grass wont shift either.
[[![Screenshot from Gyazo](https://gyazo.com/fab7550274b3064e6dc7092749ae2591/raw)](https://gyazo.com/fab7550274b3064e6dc7092749ae2591)](url)

**Comment by SamuraiShogun1102** [comment]
Better foliage shaders only work with obtifine D3

**Comment by LukaPix** [comment]
will this eventually be fixed ? Also the old optifine version crashes with many mods so using it is not an option, optifne F5 should be supported...

**Comment by QuerrixTU** [comment]
It's fixed and why this issue isn't closed ?

**Comment by LukaPix** [comment]
It does not work in my 1.12.2 Modpack with the latest forge optifine and better foliage.
On modded leaves altough, the vanilla ones work.

_Nevermind didn't reference classes of other modded leaves correctly._
